<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OBSBasicSettings</class>
 <widget class="QDialog" name="OBSBasicSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>602</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item alignment="Qt::AlignLeft">
      <widget class="QListWidget" name="listWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>135</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="iconSize">
        <size>
         <width>48</width>
         <height>48</height>
        </size>
       </property>
       <item>
        <property name="text">
         <string>General</string>
        </property>
        <property name="icon">
         <iconset resource="obs.qrc">
          <normaloff>:/settings/images/settings/system-settings-3.png</normaloff>:/settings/images/settings/system-settings-3.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Outputs</string>
        </property>
        <property name="icon">
         <iconset resource="obs.qrc">
          <normaloff>:/settings/images/settings/network-bluetooth.png</normaloff>:/settings/images/settings/network-bluetooth.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Audio</string>
        </property>
        <property name="icon">
         <iconset resource="obs.qrc">
          <normaloff>:/settings/images/settings/decibel_audio_player.png</normaloff>:/settings/images/settings/decibel_audio_player.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Video</string>
        </property>
        <property name="icon">
         <iconset resource="obs.qrc">
          <normaloff>:/settings/images/settings/video-display-3.png</normaloff>:/settings/images/settings/video-display-3.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="settingsPages">
       <property name="currentIndex">
        <number>0</number>
       </property>
       <widget class="QWidget" name="generalPage">
        <layout class="QFormLayout" name="formLayout_2">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="language"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>150</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Language:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="outputsPage"/>
       <widget class="QWidget" name="audioPage">
        <layout class="QFormLayout" name="formLayout">
         <property name="fieldGrowthPolicy">
          <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
         </property>
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_14">
           <property name="text">
            <string>Sample Rate:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="sampleRate">
           <property name="currentText">
            <string notr="true">44.1khz</string>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>22.05khz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>44.1khz</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>48khz</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>Channels:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="channelSetup">
           <property name="currentText">
            <string>Stereo</string>
           </property>
           <property name="currentIndex">
            <number>1</number>
           </property>
           <item>
            <property name="text">
             <string>Mono</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Stereo</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Desktop Audio Device 1:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="desktopAudioDevice1">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Desktop Audio Device 2:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="desktopAudioDevice2">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Mic/Aux Audio Device 1:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="auxAudioDevice1">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Mic/Aux Audio Device 2:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="auxAudioDevice2">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Mic/Aux Audio Device 3:</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="auxAudioDevice3">
           <property name="enabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QSpinBox" name="audioBufferingTime">
           <property name="minimum">
            <number>60</number>
           </property>
           <property name="maximum">
            <number>20000</number>
           </property>
           <property name="value">
            <number>700</number>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Audio Buffering Time (milliseconds):</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="audioMsg">
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 4);</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="videoPage">
        <layout class="QFormLayout" name="formLayout_3">
         <property name="labelAlignment">
          <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Renderer:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="renderer">
           <property name="currentText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="minimumSize">
            <size>
             <width>170</width>
             <height>0</height>
            </size>
           </property>
           <property name="text">
            <string>Video Adapter:</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="adapter">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="currentText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Base Resolution:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="baseResolution">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string notr="true"/>
           </property>
           <property name="duplicatesEnabled">
            <bool>false</bool>
           </property>
           <property name="frame">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Output Resolution:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="outputResolution">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="currentText">
            <string notr="true"/>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="downscaleFilter">
           <property name="enabled">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Downscale Filter:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QComboBox" name="fpsType">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="currentText">
            <string>FPS (Common)</string>
           </property>
           <property name="sizeAdjustPolicy">
            <enum>QComboBox::AdjustToContents</enum>
           </property>
           <item>
            <property name="text">
             <string>FPS (Common)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FPS (Integer)</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>FPS (Fractional)</string>
            </property>
           </item>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QStackedWidget" name="fpsTypes">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="QWidget" name="page">
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QComboBox" name="fpsCommon">
               <property name="currentText">
                <string notr="true">30</string>
               </property>
               <property name="currentIndex">
                <number>3</number>
               </property>
               <item>
                <property name="text">
                 <string notr="true">10</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">20</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">29.97</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">30</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">48</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">59.94</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string notr="true">60</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_3">
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item alignment="Qt::AlignTop">
              <widget class="QSpinBox" name="fpsInteger">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>120</number>
               </property>
               <property name="value">
                <number>30</number>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
           <widget class="QWidget" name="page_2">
            <layout class="QFormLayout" name="formLayout_4">
             <property name="labelAlignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
             <property name="leftMargin">
              <number>0</number>
             </property>
             <property name="topMargin">
              <number>0</number>
             </property>
             <property name="rightMargin">
              <number>0</number>
             </property>
             <property name="bottomMargin">
              <number>0</number>
             </property>
             <item row="0" column="1">
              <widget class="QSpinBox" name="fpsNumerator">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
               <property name="value">
                <number>30</number>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QSpinBox" name="fpsDenominator">
               <property name="minimum">
                <number>1</number>
               </property>
               <property name="maximum">
                <number>1000000</number>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Numerator:</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>Denomniator:</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="videoMsg">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(255, 0, 4);</string>
           </property>
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listWidget</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>language</tabstop>
 </tabstops>
 <resources>
  <include location="obs.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>listWidget</sender>
   <signal>currentRowChanged(int)</signal>
   <receiver>settingsPages</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>50</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>fpsType</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>fpsTypes</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>286</x>
     <y>155</y>
    </hint>
    <hint type="destinationlabel">
     <x>340</x>
     <y>154</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
